src/clr-angular/utils/host-wrapping/host-wrapping.spec.ts

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component, Directive, InjectionToken, Injector, NgModule, OnInit, Type, ViewContainerRef} from "@angular/core";
+import {ComponentFixture, TestBed} from "@angular/core/testing";
+import {By} from "@angular/platform-browser";
+
+import {DynamicWrapper} from "./dynamic-wrapper";
+import {HostWrapper} from "./host-wrapper";
+import {ClrHostWrappingModule} from "./host-wrapping.module";
+
+
+const WRAPPER_SERVICE = new InjectionToken<number>("WrapperService");
+
+@Component({
+    selector: "wrapper-component",
+    template: `Hello <ng-content></ng-content>!`,
+    providers: [{provide: WRAPPER_SERVICE, useValue: 42}]
+})
+class WrapperComponent implements DynamicWrapper {
+    _dynamic = false;
+}
+
+@Directive({selector: "[wrappedDirective]"})
+class WrappedDirective implements OnInit {
+    injected: number;
+
+    constructor(private vcr: ViewContainerRef) {}
+
+    ngOnInit() {
+        this.injected = new HostWrapper(WrapperComponent, this.vcr).get(WRAPPER_SERVICE);
+    }
+}
+
+@NgModule({
+    declarations: [WrapperComponent, WrappedDirective],
+    exports: [WrapperComponent, WrappedDirective],
+    entryComponents: [WrapperComponent]
+})
+class HostWrappingTestModule {}
+
+
+@Component({
+    template: `
+        <span wrappedDirective>World</span>
+    `
+})
+class WrappingTest {}
+
+@Component({
+    template: `
+        <wrapper-component><span wrappedDirective>World</span></wrapper-component>
+    `
+})
+class ExplicitTest {}
+
+@Component({
+    template: `
+        <span *ngIf="show" wrappedDirective>World</span>
+    `
+})
+class NgIfTest {
+    show = true;
+}
+
+
+interface TestContext<T extends WrappingTest|ExplicitTest|NgIfTest> {
+    fixture: ComponentFixture<T>;
+    wrapper: WrapperComponent;
+    wrapped: WrappedDirective;
+}
+
+fdescribe("Host wrapping", function() {
```

Comment:
Wanted to remind to change this back to `describe`

Reply:
Oops, my bad. Thanks. ❤️